app/services/collection_creator.rb

```diff
@@ -3,9 +3,21 @@ class CollectionCreator
   def self.find_or_create_collection(collection_name, unit_name, collection_desc, manager_user_key)
     # collection name is stored downcased in solr
     # TODO: check normalization method that fills in name_uniq_si field on Collection create
+
+    Rails.logger.info "XXXXXX PARAMS - #{collection_name} #{unit_name} #{collection_desc} #{manager_user_key}"
```

Comment:
Does it make sense to get rid of these logging statements and assigning the Collection to the c var?